eslint.config.js

```diff
@@ -15,7 +15,7 @@ export default [
     ...eslintConfigESLint,
     eslintConfigESLintFormatting,
     {
-        files: ["packages/espree/tests/lib/**"],
+        files: ["packages/espree/tests/lib/**", "packages/eslint-visitor-keys/tests/lib/**"],
```

Comment:
Maybe we can simplify?
```suggestion
        files: ["packages/*/tests/lib/**"],
```

Reply:
Yes, updated.